connect-example/src/main/res/drawable/ic_material_palette.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Nice! Where did you find this?

Reply:
official page: https://fonts.google.com/icons